### PR TITLE
Handle WHAT/INSTANCE_PREFIX env var mapping better

### PR DESCRIPTION
**Pull request by dobbymoodge**

### CODE REVIEW

File: cluster/test-deploy/Makefile [hunk 1]
```diff
@@ -16,8 +18,8 @@ sh: check-cluster
 .PHONY: sh
 
 check-cluster:
-ifndef WHAT
-	$(error Must set WHAT= to a unique cluster name)
+ifndef INSTANCE_PREFIX
```
Comment (smarterclayton) [inline cluster/test-deploy/Makefile]: I don't want to do this here, it's inconsistent with other playbooks.  We should always set INSTANCE_PREFIX to WHAT.

Comment (dobbymoodge) [reply cluster/test-deploy/Makefile]: Reverted this change.

**Comment by dobbymoodge** [comment]
@smarterclayton updated `cluster/ci/Makefile` as well

**Comment by smarterclayton** [comment]
Maybe I wasn’t clear - I don’t want the variable to be conditional, it has to be set in those jobs to a value.  Don’t make setting it conditional.

**Comment by smarterclayton** [comment]
No, reread (though it was same file).

**Comment by smarterclayton** [comment]
Nm

File: cluster/ci/Makefile [hunk 2]
```diff
@@ -1,7 +1,9 @@
 LATEST = $(shell curl -q "https://storage.googleapis.com/origin-ci-test/branch-logs/origin/master/builds/.latest" 2>/dev/null)/artifacts/rpms
 REPO ?= $(LATEST)
-INSTANCE_PREFIX ?= $(WHAT)
```
Comment (smarterclayton) [inline cluster/ci/Makefile]: Is there a reason we can’t make this an unconditional equality?  Ie remove the “?”

Comment (dobbymoodge) [reply cluster/ci/Makefile]: ?? That's what this PR does ??

Comment (dobbymoodge) [reply cluster/ci/Makefile]: @smarterclayton Any chance getting some clarification on this comment?

Comment (dobbymoodge) [reply cluster/ci/Makefile]: @smarterclayton This is needed for me to automate pruning, ptal and merge

**Pull request closed without merging**